### PR TITLE
change route name from camelcase to dasherize

### PR DESCRIPTION
**Pull request by dianafa**
WIP. Do not merge. Needs to be tested before.

Make infoboxbuilder route to fit routing naming conventions proposed here: https://github.com/Wikia/guidelines/pull/106

## Links

https://github.com/Wikia/guidelines/pull/106

## Reviewers

@RafLeszczynski @bognix @vforge

### CODE REVIEW

File: front/main/app/router.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const Router = Ember.Router.extend({
 Router.map(function () {
 	const articlePath = '/wiki/';
 
-	this.route('article-preview', {
+	this.route('articlePreview', {
```
Comment (hakubo) [inline front/main/app/router.js]: '/article-preview' will be default for articlePreview so you don't have to specify path

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Comment by rogatty** [comment]
>Make all routes to fit routing naming conventions

If this is still valid then would be nice to update _all_ of them.

**Comment by dianafa** [comment]
@igor This would require HUGE regression testing. Let's start with small steps.

**Comment by dianafa** [comment]
Tested. Works. Merging.

**Pull request merged**